azext_iot/central/params.py

```diff
@@ -654,3 +661,261 @@ def load_central_arguments(self, _):
             options_list=["--children-ids"],
             help="Space-separated list of children device ids.",
         )
+
+    with self.argument_context("iot central enrollment-group") as context:
+        context.argument(
+            "group_id",
+            options_list=["--group-id", "--id"],
+            help="Unique identifier for the enrollment group.",
+        )
+        context.argument(
+            "entry",
+            options_list=["--entry"],
+            help="Entry of certificate only support primary and secondary.",
+        )
+
+    with self.argument_context("iot central enrollment-group create") as context:
+        context.argument(
+            "attestation",
+            options_list=["--attestation", "--at"],
+            help="The attestation mechanism for the enrollment group. Provide path to JSON file or raw stringified JSON."
+            " [File Path Example:./path/to/attestation.json]"
+            " [Example of stringified JSON:[{<Attestation Data JSON>}]."
+            " The request body must contain GroupSymmetricKeyAttestation or GroupX509Attestation.",
+        )
+        context.argument(
+            "display_name",
+            options_list=["--display-name"],
+            help="Display name of the enrollment group."
+        )
+        context.argument(
+            "type",
+            options_list=["--type"],
+            help="Type of devices that connect through the group."
+        )
+        context.argument(
+            "enabled",
+            options_list=["--enabled"],
+            arg_type=get_three_state_flag(),
+            help="Whether the devices using the group are allowed to connect to IoT Central. True or False."
+        )
+        context.argument(
+            "certificate",
+            options_list=["--certificate", "--cert"],
+            help="The string representation of this x509 certificate.",
+        )
+        context.argument(
+            "verified",
+            options_list=["--verified", "--v"],
+            arg_type=get_three_state_flag(),
+            help="Whether the x509 certificate has been verified. True or False.",
```

Comment:
does this flag actually do something? (cause I have seen other commands where this flag does nothing and you have to go through the generate verification code and upload verification cert to actually verify the cert)

Reply:
also the arg type should handle what options are allowed so you should not need the True or False.

Reply:
Because iirc users can set an unverified cert to enrollment group

Reply:
Removed.